Mathlib/RingTheory/Ideal/Norm/RelNorm.lean

```diff
@@ -300,20 +300,35 @@ theorem map_relNorm (I : Ideal S) {T : Type*} [Semiring T] (f : R →+* T) :
 theorem relNorm_mono {I J : Ideal S} (h : I ≤ J) : relNorm R I ≤ relNorm R J :=
   spanNorm_mono R h
 
-open MulSemiringAction Pointwise in
+variable {R}
+
+/--
+See `relNorm_map_algEquiv` for a version that shows equality between both sides.
+-/
+theorem relNorm_map_algEquiv_le {T : Type*} [CommRing T] [IsDedekindDomain T] [IsIntegrallyClosed T]
+    [Algebra R T] [Module.Finite R T] [NoZeroSMulDivisors R T]
+    [Algebra.IsSeparable (FractionRing R) (FractionRing T)] (σ : S ≃ₐ[R] T) (I : Ideal S) :
+    relNorm R (I.map σ) ≤ relNorm R I :=
+  span_mono fun _ ⟨x, hx₁, hx₂⟩ ↦ ⟨σ.toRingEquiv.symm x,
+    by rwa [SetLike.mem_coe, Ideal.symm_apply_mem_of_equiv_iff],
+    hx₂ ▸ Algebra.intNorm_map_algEquiv _ x σ.symm⟩
```

Comment:
This is strictly weaker than the lemma below, right? If so, it should probably just be `private`

Reply:
I am not sure that the case since we need also to make sure that `intNorm` value stays the same and AFAIK we only have this result for AlgEquiv.